tests/resources/xml/my-document.xmi

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><xmi:XMI xmlns:xmi="http://www.omg.org/XMI" xmlns:tcas="http:///uima/tcas.ecore" xmlns:cas="http:///uima/cas.ecore" xmi:version="2.0"><cas:NULL xmi:id="0"/><tcas:DocumentAnnotation xmi:id="2" sofa="1" begin="0" end="77" language=""/><cas:Sofa xmi:id="1" sofaNum="1" sofaID="_InitialView" mimeType="text" sofaString="This is a text, put in a Cas. Could have some annotations, doesn't though :(&#10;"/><cas:View sofa="1" members="2"/></xmi:XMI>
```

Comment:
Please format - or even better, do not have the file at all but instead create it dynamically as part of the test.